File: tests/grader/fixtures/problems_for_verification_tests/hworld/solutions/rose-ruby-f/hworld.rb
def main()
    input_filename = ARGV.length == 0 ? "hworld.in" : ARGV[0]
    output_filename = ARGV.length == 0 ? "hworld.out" : ARGV[1]

    count = read_input(input_filename)
    lines = solve(count)
    write_output(output_filename, lines)
end


def read_input(filename)
    return File.read(filename).to_i
end


def solve(count)
    return ["Hello world!"] * (count + 1)
end


def write_output(filename, lines)
    File.write(filename, lines.join("\n"))
end


main()

File: tests/grader/fixtures/problems_for_language_tests/hworld/solutions/peter-python/hworld.py
import sys


def main():
    input_filename = "hworld.in" if len(sys.argv) == 1 else sys.argv[1]
    output_filename = "hworld.out" if len(sys.argv) == 1 else sys.argv[2]

    count = read_input(input_filename)
    lines = solve(count)
    write_output(output_filename, lines)


def read_input(filename):
    with open(filename) as input_file:
        count = int(input_file.readline())
        return count


def solve(count):
    return ["Hello world!" for _ in range(0, count)]


def write_output(filename, lines):
    with open(filename, "w") as output_file:
        output_file.write("\n".join(lines))


if __name__ == "__main__":
    main()
